packages/core-mobile/app/new/routes/onboarding/keystone/recoveryUsingKeystone.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { useRouter } from 'expo-router'
+import KeytoneSDK, { UR } from '@keystonehq/keystone-sdk'
+import { RecoveryUsingKeystone as Component } from 'features/onboarding/components/RecoveryUsingKeystone'
+import Logger from 'utils/Logger'
+import { fromPublicKey } from 'bip32'
+import KeystoneService from 'hardware/services/KeystoneService'
+
+export default function RecoveryUsingKeystone(): JSX.Element {
+  const { navigate, replace } = useRouter()
+
+  function handleNext(ur: UR): void {
+    try {
+      const sdk = new KeytoneSDK()
+      const accounts = sdk.parseMultiAccounts(ur)
+      const mfp = accounts.masterFingerprint
+      const ethAccount = accounts.keys.find(key => key.chain === 'ETH')
+      const avaxAccount = accounts.keys.find(key => key.chain === 'AVAX')
+      if (!ethAccount || !avaxAccount) {
+        throw new Error('No ETH or AVAX account found')
+      }
```

Comment:
could this be moved inside KeystoneService.init()? that way, only the service knows about `@keystonehq/keystone-sdk` dependency